indico/modules/events/editing/templates/editing.html

```diff
@@ -5,18 +5,12 @@
 {% endblock %}
 
 {% block content %}
-    <div id="file-manager"
-         data-event-id="{{ event.id }}"
+    <div data-event-id="{{ event.id }}"
+         data-contribution-id="{{ editable.contribution.id }}"
+         data-editable-type="{{ editable.type.name }}"
+         id="editing-timeline"
          data-download-url="{{ url_for('event_editing.revision_files_export', event, contrib_id=contribution.id,
```

Comment:
Pretty sure this one should go away since it's hardcoded to download the first revision for a paper